ui/src/api-logic/axios/api.ts

```diff
@@ -17,3 +17,31 @@ export const validateGtihubToken = async (pat: string) => {
     return false
   }
 }
+
+/**
+ * Method to login to Postgrahile
+ * @param user Database user
+ * @param password Database password
+ * @returns true if it is logged in, otherwise returns false
+ */
+export const auth = async (user: string, password: string) => {
+  try {
+    const response = await axios.post('/api/admin-auth', { user, password })
+    return response.status === 200
+  } catch (error) {
+    return false
```

Comment:
we should handle the `error` here in some way, even if it's just logging it out to the console for now (maybe display in an alert if we can?)

just so we dont fail silently for now

Reply:
Done [65cc09f](https://github.com/mergestat/fuse/pull/334/commits/65cc09f4eb0c07e3680c9002f5e745931c2ca6b9)